testing/integration_tests/test_graph.py

```diff
@@ -0,0 +1,69 @@
+import pytest, json
+from src.utils.graph import Graph
+import networkx as nx
+
+# Load sample response data
+# This file path might break with automated testing
+with open('testing/integration_tests/test_response.json') as f:
+    sample_data = json.load(f)
+
+# Create a truth graph to compare
+known_graph = nx.DiGraph()
+start = 'Boston%2CMA'
+end = 'Lexington%2CMA'
+se_node = 'Boston%2CMA&Lexington%2CMA'
+midpoints = ['Charleston%2CMA', 'Concord%2CMA', 'Sudbury%2CMA', 'Westwood%2CMA', 'Arlington%2CMA', 'Needham%2CMA']
+origins = [start] + midpoints
+dests = [end] + midpoints
+
+def test_graph():
```

Comment:
I've been putting docstrings before my tests - is that something we want to do?

Reply:
yes